src/Examine.Lucene/Providers/LuceneSearcher.cs

```diff
@@ -51,8 +51,23 @@ public LuceneSearcher(string name, SearcherManager searcherManager, Analyzer ana
         public override ISearchContext GetSearchContext()
             => new SearchContext(_searcherManager, _fieldValueTypeCollection);
 
+        //protected override void Dispose(bool disposing)
+        //{
+        //if (!_disposedValue)
+        //{
+        //    if (disposing)
+        //    {
+        //        _searcherManager.Dispose();
+        //    }
+
+        //    _disposedValue = true;
+        //}
+        //base.Dispose(disposing);
+        //}
```

Comment:
This will replace the `new virtual` when possible to align with the dispose pattern.

Reply:
Looks like this would break any classes overriding public virtual void Dispose(). We should revert to keep compatibility and mark as obsolete in V5.

Reply:
This code is commented out I can't see it being problematic before uncommented?